res/sections/20-WorkingEnvironment.tex

```diff
@@ -0,0 +1,267 @@
+\chapter{Ambiente di Lavoro}
+
+Per far funzionare \LaTeX{} sul vostro computer dovete scaricare anche un
+compilatore, che potete trovare a questo all'indirizzo
+\url{https://miktex.org/download}. L'installazione è facile ma il completamento
+potrebbe richiedere un po' di tempo. Siate pazienti.
+Per gli scopi di questo corso utilizzeremo \textbf{TeXnicCenter}, un comodo IDE
+che potete scaricare gratuitamente all'indirizzo
+\url{http://www.texniccenter.org/download/}.
+Ovviamente avrete bisogno anche di un visualizzatore PDF, in questo caso noi
+consigliamo un lettore PDF leggero e veloce (cosa che Adobe Reader non è), che
+ci sarà molto utile in quanto dovremo aprirlo ripetutamente: il lettore PDF in
+questione si chiama SumatraPDF ed è gratuitamente scaricabile al seguente
+indirizzo \url{https://www.sumatrapdfreader.org/download-free-pdf-viewer.html}.
+In ogni caso siete liberi di utilizzare il lettore PDF che più preferite.
+
+\section{Windows e TeXnicCenter}
+
+Durante la prefazione (che nessuno di voi avrà certamente letto) abbiamo
+parlato di IDE. Per il nostro corso ne utilizzeremo uno (come già detto sopra)
+e in questo capitolo vedremo un attimo le sue funzionalità, giusto per prendere
+mano con l'ambiente che dovremo usare.
+
+Se avete installato TeXnicCenter sul vostro computer per la prima volta e lo
+state aprendo solo ora vi chiederà alcune informazioni riguardo al lettore DVI
+e PDF da usare. Riguardo al lettore DVI potete pure lasciare i campi vuoti,
+mentre per il lettore PDF premete i tre puntini accando alla prima casella di
+testo, e selezionate nel vostro hard disk l'eseguibile del lettore PDF che più
+preferite.
+
+Prima di tutto, alla apertura del programma ci troveremo ad una schermata
+vuota, in cui possiamo vedere diversi bottoni.
+
+\begin{figure}[H]
+  \centering
+  \includegraphics[scale=0.35]{texniccenterprincipale}
+  \caption{Schermata principale di TeXnicCenter}
+  \label{img:principale_texcenter}
+\end{figure}
+
+Come possiamo vedere dall'immagine\footnote{Sappiamo che si vede poco
+ma questo è il meglio che siamo riusciti a fare} notiamo partendo da in alto
+a sinistra e scorrendo verso destra che inizialmente troviamo dei bottoni per
+gestire il progetto (o per crearne di nuovi).
+Abbiamo subito dopo le opzioni per la compilazione: il menù a tendina ci
+permette di selezionare che tipo di output vogliamo (se PDF o altri formati) e
+i bottoni accanto a questo menù ci permettono di lanciare la compilazione
+del progetto.
+Nella seconda riga troviamo le impostazioni per aggiungere elenchi puntati e
+numerati, seguiti dai bottoni per aggiungere immagini e tabelle. Nel penultimo
+riquadro ci sono i classici bottoni per impostare gli stili di formattazione
+(quelli che siamo abituati ad avere in Word per intenderci).
+Alla fine troviamo i bottoni per l'inserimento di formule matematiche, che
```

Comment:
Dici qui che le formule matematiche le vedremo in seguito. Dicevo solo di essere sicuri che questo succeda